### DISCUSSION TITLE
Using arrow keys with embedded player wrapper

### DISCUSSION THREAD

**Discussion by danaburger** [vimeo/player.js#980] [Q&A] [answered]
"I am building a wrapper for the embedded player for use in sports analysis. One of the things I need to do is provided some different keyboard controls than the ones that are built into the player. For example, I need ArrowLeft and ArrowRight events to step a single frame. I can program my wrapper to do this in my browser with the exception of when the player itself has focus. I can't get an overlay <div> with a high z-index to cover the iframe so that it intercepts events in the capture phase, and stops their downward propagation if I don't what the player itself to hand the event.

Is the element in the player that handles keyboard events positioned statically, so that it doesn't respect z-index? Can you suggest how I can solve this? Is there some way to program the key handles through the player API?"

**Comment by luwes** [comment, marked as answer]
I would recommend finding a way to make the player unfocusable while these custom key arrow actions are performed. some info https://stackoverflow.com/questions/716235/how-to-make-a-div-unfocusable
